### PR TITLE
Release prep (again)

### PR DESCRIPTION
**Pull request by bfriesen**

### CODE REVIEW

File: Rock.Messaging/MQ/MessageCompression.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿using System.Text;
+﻿using RockLib.Immutable;
```
Comment (JustinMoss) [inline Rock.Messaging/MQ/MessageCompression.cs]: This needs to be in a #if rocklib

Comment (bfriesen) [reply Rock.Messaging/MQ/MessageCompression.cs]: Good catch. I updated the PR with stuff that needed to be changed in order for the old solution to build.

**Comment by JustinMoss** [comment]
👍

**Comment by derikwhittaker** [comment]
No issues.  Will Merge

**Pull request merged**